code/datums/shuttles.dm

```diff
@@ -138,3 +138,8 @@
 	suffix = "armory"
 	name = "NRV Sparta"
 	description = "Armory Shuttle, with plenty of guns to hand out and some general supplies."
+
+/datum/map_template/shuttle/admin/hospital
```

Comment:
Might've forgotten to rename the typepath.

Reply:
Im really slipping

Reply:
its not just me!

Reply:
No this was tested, I just didn't realise a shuttle was missing from the list.

It didn't break the entire server.